code/game/objects/items/defib.dm

```diff
@@ -273,6 +275,21 @@
 	cell = new /obj/item/stock_parts/cell/infinite(src)
 	update_icon()
 
+/obj/item/defibrillator/compact/combat/cooldowncheck(mob/user)
+	addtimer(CALLBACK(src, .proc/finish_charging), 2.5 SECONDS)
+
+/obj/item/defibrillator/compact/combat/make_paddles()
+	return new /obj/item/twohanded/shockpaddles/syndicate(src)
+
+/obj/item/twohanded/shockpaddles/syndicate/update_icon()
+	icon_state = "syndiepaddles[wielded]"
```

Comment:
don't just copy/paste the same code over again. Just add a new variable to the paddles that defines the base icon_state